crates/pathfinder/src/sequencer/reply.rs

```diff
@@ -314,11 +314,18 @@ pub mod transaction {
             match self {
                 Transaction::Declare(t) => t.transaction_hash,
                 Transaction::Deploy(t) => t.transaction_hash,
-                Transaction::Invoke(t) => t.transaction_hash,
+                Transaction::Invoke(t) => match t {
+                    InvokeTransaction::V0(t) => t.transaction_hash,
+                    InvokeTransaction::V1(t) => t.transaction_hash,
+                },
             }
         }
     }
 
+    fn default_transaction_version() -> TransactionVersion {
+        TransactionVersion(web3::types::H256::zero())
+    }
```

Comment:
Personally, I find this a bit misleading. Is the default 0 even though that's simply.. legacy and will definitely not be the "current" default.

Have not yet reached the usage, but maybe a `TransactionVersion::ZERO` would be more appropriate?

Reply:
Hmm okay after seeing usage is to provide default serde when missing values, I would definitely go with a more literal name involving `zero`. Unsure if you can use consts with the default or if it must be a `fn`.

Reply:
Okay, I've removed `Default` for `TransactionVersion` and went with the more explicit `transaction_version_zero` function name.